### PR TITLE
Mt isa df kd

### PR DESCRIPTION
**Pull request by fourndo**
Updating the text
Suggested reviewers:

- [ ] @krisdavis 

Before merging: please run a spell check through your documents! 

Figure updates needed:

- [x]  Survey: Pseudo-sections and recovered models into 2 separate figs (with conductor on top)
- [x]  2D IP (P-DP and DP-P) inversions with chifact of 4
- [x]  2D IP  (merged) inversions with chifact of 4

### CODE REVIEW

File: content/case_histories/mt_isa/survey.rst [hunk 1]
```diff
@@ -62,23 +46,12 @@ locations (Powered by: `SimPEG <http://www.simpeg.xyz/>`_).
...
-in the inversion were:  xxxxx. The Mt. Norvit Horizon is evident.
+    Log10 pseudo-conductivity section along the geological section and the recovered 2D conductivity model from the inversion of DCR data. The top two figures are for the pole-dipole survey and the bottom two images coorrespond to the dipole-pole survey along the same line. In both figures, the true conductivity model is shown in grey scale for reference.
 
-Using 2D forward modelling, we can also simulate current flows that could be
-expected at Mt. Isa. Note in particular the current channeling into the
-conductive Breakaway Shale as well as through the deeper mineralization.
+.. todo::
+  To determine if the survey is well designed we use the criteria for :ref:`detectability <surveyDetectability>` The difference sections are shown in {figure}. They show xxx If errors are assigned as 5% + floor 0.0001 V then the secondary signal from the target is detectable.
 
+  Inversion of Synthetic data Following the work {link: general basics of survey design} we invert the synthetic data using 2D algorithm.  The parameters used in the inversion were:  xxxxx. The Mt. Norvit Horizon is evident.
```
Comment (lheagy) [inline content/case_histories/mt_isa/survey.rst]: can you put the "todo" items in a `.. todo:: ` environment (so we don't forget about them!) - same as the `.. topic::` environment

File: content/case_histories/mt_isa/setup.rst [hunk 2]
```diff
@@ -3,34 +3,19 @@
 Setup
 =====
 
-.. Prelude
-.. -------
+The Mount Isa region (:numref:`Location_map`) hosts numerous deposits (Lead, Zinc, Silver, Copper, and Gold) within an area under continuous exploration. This region is found in west Queenland, Australia. Within this region, ten lines acquiring direct current (DCR) and induced polarization (IP) lines were placed over what is now referred to as the Cluny Mine (Copper), 22 km south of the city of Mt Isa.
```
Comment (lheagy) [inline content/case_histories/mt_isa/setup.rst]: could you add links to DCR, IP portions of this case history?

File: content/case_histories/mt_isa/survey.rst [hunk 3]
```diff
@@ -3,42 +3,27 @@
...
+(b) The area of interest is approximately 2km by 5km. Although full 3D coverage was desireable, the field acquistion was limited to 10 east-west lines. The reason for this was two fold. Firstly, the 2D lines could be laid out across the East-West boundaries of Cluny region. Secondly, the fault structures were known to strike north-south, so it is natural to have the survey perpendicular to strike in order to generate the most physical property contrast along line. Most lines consisted of 21 current electrode locations (the three to the north had 19) with each current electrode having a maximum of 20 potential readings.
 
-.. _MIMDAS: http://www.smedg.org.au/Sym01NS.htm
+(c) The choice of a pole-dipole was motivated by past experience that this was an effective survey for deep targets. The layout shown in {figure} indicates a current electrode at position 1 with potentials measured across all other dipoles. This produces a :ref:`pseudo-section<dcr_pseudosection>`. Furthermore, with the MIMDAS system, a pole-dipole (P-DP) and dipole-pole (DP-P) could be acquired along each line at no additional cost. The system spaced each potential electrode 100-m apart.
```
Comment (lheagy) [inline content/case_histories/mt_isa/survey.rst]: do you have a link for the referenced figure?

**Comment by fourndo** [comment]
Leave branch open for later update

**Pull request merged**